lms/djangoapps/course_blocks/transformers/helpers.py

```diff
@@ -0,0 +1,38 @@
+"""
+Transformers helpers functions.
+"""
+from openedx.core.djangoapps.user_api.partition_schemes import RandomUserPartitionScheme
+from openedx.core.djangoapps.course_groups.partition_scheme import CohortPartitionScheme
+# TODO 8874: Make it so we support all schemes instead of manually declaring them here.
+INCLUDE_SCHEMES = [CohortPartitionScheme, RandomUserPartitionScheme, ]
```

Comment:
Why are we supporting only a subset of the partition schemes? `has_access` is not exclusive about this. And we plan to replace `has_access` with the transformers implementation.

For example, there is now a new `VerificationPartitionScheme` that has been added, which we need to support.
Can we remove the `INCLUDE_SCHEMES` list and just support all schemes as is done in `has_access`?